http/digest/src/test/java/org/wildfly/security/http/digest/DigestMechanismFactoryTest.java

```diff
@@ -0,0 +1,83 @@
+package org.wildfly.security.http.digest;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.wildfly.security.http.HttpAuthenticationException;
+import org.wildfly.security.http.HttpServerAuthenticationMechanism;
+
+import javax.security.auth.callback.Callback;
+import javax.security.auth.callback.CallbackHandler;
+import javax.security.auth.callback.UnsupportedCallbackException;
+import java.io.IOException;
+import java.util.HashMap;
+
+public class DigestMechanismFactoryTest {
+
+    private DigestMechanismFactory digestMechanismFactory = new DigestMechanismFactory();
+
+    CallbackHandler dummyCallbackHandler = new CallbackHandler() {
+        @Override
+        public void handle(Callback[] callbacks) throws IOException, UnsupportedCallbackException {
+        }
+    };
+
+    @Test
+    public void testGetMechanismNamesPropertiesNull(){
+        digestMechanismFactory.getMechanismNames(null);
+        digestMechanismFactory.getMechanismNames(new HashMap<String,String>());
+    }
+
+    @Test
+    public void testGetMechanismNamesReturnNotNull(){
+        String[] mechanismNames = digestMechanismFactory.getMechanismNames(null);
+        Assert.assertNotNull("Array of mechanism names is not null.", mechanismNames);
```

Comment:
s/"Array of mechanism names is not null."/"Array of mechanism names cannot be null."

Reply:
@keshav-redhat This was not updated